reporter.go

```diff
@@ -461,12 +465,16 @@ func (r *Reporter) PrintDiffSummary(out io.Writer, report *Report) {
 		fmt.Fprintln(out)
 	}
 	if len(report.Errors) > 0 {
+		changes = true
 		fmt.Fprintf(out, "Reporting Errors (%d):\n", len(report.Errors))
 		for _, file := range report.Errors {
 			fmt.Fprintf(out, "%s: %v\n", file.Before.Path, file.Err)
 		}
 		fmt.Fprintln(out)
 	}
+	if !changes {
```

Comment:
Wouldn't it be nice if there were a report method which tells you whether there are any changes. For example:

```go
// Blank reports whether the report is empty.
// The report is empty if it has no additions, no deletions, no modifications and no errors.
func (rep *Report) Blank() bool {
  return len(rep.Added) + len(rep.Deleted) + len(rep.Modified) + len(rep.Errors) == 0
}
```

Unsure about the name. Some alternatives: `Any`, `Empty`.

Reply:
Yep, it makes sense. I like `Empty` for the name though.